From 7eb97b37768e404b3a919b2d85162943c4ed9ca3 Mon Sep 17 00:00:00 2001
From: Martchus <martchus@gmx.net>
Date: Tue, 1 Jun 2021 13:00:33 +0200
Subject: [PATCH 32/32] Add missing `#include <limits>` in QOffsetStringArray
 header

At least with GCC 10/11 targeting mingw-w64 the compilation
fails otherwise.

Change-Id: Icad20343a58f9aa1293ba443bebf70373e7dc762
---
 src/corelib/tools/qoffsetstringarray_p.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/corelib/tools/qoffsetstringarray_p.h b/src/corelib/tools/qoffsetstringarray_p.h
index 4dd9e9603b..e26a57ff43 100644
--- a/src/corelib/tools/qoffsetstringarray_p.h
+++ b/src/corelib/tools/qoffsetstringarray_p.h
@@ -55,6 +55,7 @@
 
 #include <tuple>
 #include <array>
+#include <limits>
 
 QT_BEGIN_NAMESPACE
 
-- 
2.31.1

